packages/devtools_app/lib/src/flutter/app.dart

```diff
@@ -82,31 +102,53 @@ class DevToolsAppState extends State<DevToolsApp> {
   }
 
   /// The routes that the app exposes.
-  final Map<String, UrlParametersBuilder> _routes = {
-    '/': (_, params) => Initializer(
-          url: params['uri'],
-          builder: (_) => DevToolsScaffold(
-            tabs: const [
-              InspectorScreen(),
-              TimelineScreen(),
-              MemoryScreen(),
-              PerformanceScreen(),
-              DebuggerScreen(),
-              if (showNetworkPage) NetworkScreen(),
-              LoggingScreen(),
-              InfoScreen(),
-            ],
-            actions: [
-              HotReloadButton(),
-              HotRestartButton(),
-              OpenSettingsAction(),
-              OpenAboutAction(),
-            ],
-          ),
-        ),
-    '/connect': (_, __) =>
-        DevToolsScaffold.withChild(child: ConnectScreenBody()),
-  };
+  Map<String, UrlParametersBuilder> get routes => _routes ??= {
+        '/': (_, params) => Initializer(
+              url: params['uri'],
+              builder: (_) => DevToolsScaffold(
+                tabs: [
+                  const InspectorScreen(),
+                  const TimelineScreen(),
+                  const MemoryScreen(),
+                  const PerformanceScreen(),
+                  const DebuggerScreen(),
+                  if (showNetworkPage) const NetworkScreen(),
+                  const LoggingScreen(),
+                  const InfoScreen(),
+                  ..._generateConditionalScreens(),
```

Comment:
We may want to have some of the always included screens at the end of the list (logging? info?). Perhaps put the conditional screens ahead of the logging screen entry.